tests/test_declarations/angular_async.js

```diff
@@ -14,4 +14,9 @@ it("does something really long and complicated so I have to write a very long na
   // code
 }));
 
+/*
+* isTestCall(parent) should only be called when parent exists
+* and parent.type is CallExpression. isTestCall(parent) will
+* not be called for the below test
```

Comment:
We should mention that this is mainly for testing if it throw errors.

Reply:
done https://github.com/prettier/prettier/pull/4241/commits/a6ffc4ca66d7fb1e65568a387667df188c938468